### PR TITLE
SorrTask-891 Unit tests for compute epoch function

### PR DESCRIPTION
**Pull request by aish-nidhi**
Task [#891](https://github.com/kaizen-ai/kaizenflow/issues/891)

- Added unit tests for compute_epoch function:-
   -> Function for fetching test data, one each for series and dataframe
   -> 3 Unit Tests, one for each available unit value for Series input
   -> Unit test with dataframe input

### CODE REVIEW

**Comment by DanilYachmenev** [comment]
@aish-nidhi while your PR is being reviewed, read  carefully https://github.com/kaizen-ai/kaizenflow/blob/master/docs/coding/all.submit_code_for_review.how_to_guide.md and update PR accordingly

keep in mind that in your probation term the most important part that we look at is your ability to follow the specs and our code style
tasks are basic, what is expected is ability to understand documentation and do what is requested so we do not need to spend time on additional explainations

**Comment by DanilYachmenev** [comment]
Also pls remove a TODO above the func def
Снимок экрана 2024-05-03 в 00 18 22

**Comment by aish-nidhi** [comment]
Resolved comments and updated PR. Also, removed the TODO comment above the function.

File: core/finance/test/test_prediction_processing.py [hunk 1]
```diff
@@ -58,3 +58,141 @@ def _get_data() -> pd.DataFrame:
...
+        self.check_srs_output(
+            result, expected_length, expected_column_value, expected_signature
+        )
+
+    def test4(self) -> None:
+        """
+        Check that epoch is computed correctly for dataframe input.
+        """
+        srs = self.helper()
+        # Convert series data to dataframe
```
Comment (DanilYachmenev) [inline core/finance/test/test_prediction_processing.py:170]: not needed, self-explanatory

File: core/finance/test/test_prediction_processing.py [hunk 2]
```diff
@@ -58,3 +58,141 @@ def _get_data() -> pd.DataFrame:
...
+        df = srs.to_frame()
+        # Testing input with default unit - minute.
```
Comment (DanilYachmenev) [inline core/finance/test/test_prediction_processing.py:172]: same here, ok to remove

**Review by DanilYachmenev — COMMENTED** [review]
2 nits and lgtm for me
ping @samarth9008 for a final review when done

**Review by samarth9008 — COMMENTED** [review]
LG

**Pull request merged**